modules/monitoring-setting-scrape-and-evaluation-intervals-limits-for-user-defined-projects.adoc

```diff
@@ -46,31 +52,23 @@ metadata:
 data:
   config.yaml: |
     prometheus:
-      enforcedSampleLimit: 50000 <1>
+      enforcedSampleLimit: 50000 # <1>
+      enforcedLabelLimit: 500 # <2>
+      enforcedLabelNameLengthLimit: 50 # <3>
+      enforcedLabelValueLengthLimit: 600 # <4>
+      scrapeInterval: 1m30s # <5>
+      evaluationInterval: 1m15s # <6>
 ----
 <1> A value is required if this parameter is specified. This `enforcedSampleLimit` example limits the number of samples that can be accepted per target scrape in user-defined projects to 50,000.
-
-. Add the `enforcedLabelLimit`, `enforcedLabelNameLengthLimit`, and `enforcedLabelValueLengthLimit` configurations to `data/config.yaml` to limit the number of scraped labels, the length of label names, and the length of label values in user-defined projects:
-+
-[source,yaml]
-----
-apiVersion: v1
-kind: ConfigMap
-metadata:
-  name: user-workload-monitoring-config
-  namespace: openshift-user-workload-monitoring
-data:
-  config.yaml: |
-    prometheus:
-      enforcedLabelLimit: 500 <1>
-      enforcedLabelNameLengthLimit: 50 <2>
-      enforcedLabelValueLengthLimit: 600 <3>
-----
-<1> Specifies the maximum number of labels per scrape.
+<2> Specifies the maximum number of labels per scrape.
 The default value is `0`, which specifies no limit.
-<2> Specifies the maximum length in characters of a label name.
+<3> Specifies the maximum length in characters of a label name.
 The default value is `0`, which specifies no limit.
-<3> Specifies the maximum length in characters of a label value.
+<4> Specifies the maximum length in characters of a label value.
 The default value is `0`, which specifies no limit.
+<5> Specifies the interval between consecutive scrapes. The interval must be set between 5 seconds and 5 minutes.
+The default value is `30s`.
+<6> Specifies the interval between Prometheus rule evaluations. The interval must be set between 5 seconds and 5 minutes.
```

Comment:
both UWM prometheus and ThanosRuler support evaluationInterval setting, for UWM prometheus, default value for evaluationInterval is `30s`, see from https://github.com/openshift/cluster-monitoring-operator/blob/release-4.18/Documentation/api.md#prometheusrestrictedconfig

for ThanosRuler, default value is `15s`, see from https://github.com/openshift/cluster-monitoring-operator/blob/release-4.18/Documentation/api.md#thanosrulerconfig